charts/prometheus-pgbouncer-exporter/values.yaml

```diff
@@ -0,0 +1,221 @@
+# Default values for prometheus-pgbouncer-exporter.
+
+# Also deploy the postgresql helm chart dependency
+postgresql:
```

Comment:
i would remove this entirely as it's only for ci.

Reply:
done, thanks!